### ISSUE TITLE
@types Definitions for leaflet-groupedlayercontrol Feature Request

### ISSUE THREAD

**Issue by marcus343a** [ismyrnow/Leaflet.groupedlayercontrol#65]
Would love someone to help me defined the typing's for this plugin. I've been struggling with it for a few days but haven't gotten far.  It may be because I'm trying to integrate this into a project using ngx-leaflet.

**Comment by MojioMS** [comment]
> I'm trying to integrate this into a project using ngx-leaflet.

did u get any further, @marcus343a ?

**Comment by marcus343a** [comment]
Nope. I ending up rolling my own layer manager. I needed to add opacity slider as well. 
![image](https://user-images.githubusercontent.com/2332697/50349721-c1c16680-050a-11e9-88b1-b4bd84fe11ad.png)
![image](https://user-images.githubusercontent.com/2332697/50349730-c84fde00-050a-11e9-9ee7-cb07ec5a8344.png)

**Comment by Kernonen** [comment]
Hello 
I'll be interested in your layer opacity code. Would you share it ?
